invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -338,7 +338,7 @@ def new_record():
 def deposit_create(community=None):
     """Create a new deposit."""
     return render_template(
-        "invenio_app_rdm/records/deposit.html",
+        current_app.config["DEPOSIT_CREATE_TEMPLATE"],
```

Comment:
Do you have a use case when you want to differentiate the create from the edit template? I am hesitating to split them even If I can think of a case...But, if there is none at the moment I would keep it the same :) Maybe @kpsherva @ntarocco can comment on that too!

Reply:
I agree with @zzacharo, I would like to understand what would be the use case to separate them. I find it very dangerous due to complexity of the form, this change opens the gate for `current complexity^2` which has consequences for maintainability and increases the risk of possible unwanted outcomes

Reply:
One use case that I might think of off on top of my head, is if I want to show particular text that should not be displayed on the other template, yes we can create an if statement, but reducing coupling is also a valid point when we look for flexibility in my opinion.

Reply:
I agree with the flexibility option, but I would enable it when needed than from the beginning as it adds on maintenance complexitiy! It's an easy change to add afterward when an actual use case would be needed!